com.unity.render-pipelines.universal/Runtime/Passes/InvokeOnRenderObjectCallbackPass.cs

```diff
@@ -10,6 +10,7 @@ public InvokeOnRenderObjectCallbackPass(RenderPassEvent evt)
         {
             base.profilingSampler = new ProfilingSampler(nameof(InvokeOnRenderObjectCallbackPass));
             renderPassEvent = evt;
+            useNativeRenderPass = false;
```

Comment:
we should note this as a TODO, as technically this shouldn't be breaking RenderPass